chart/kubeapps/values.yaml

```diff
@@ -12,6 +12,10 @@
 ## If you set it to true, Kubeapps will not work with releases installed with Helm 2.
 useHelm3: false
 
+## Enable this feature flag to allow users to discover available namespaces (only the ones they have access).
+## If you set it to true, Kubeapps create a ClusterRole to be able to list namespaces.
+allowNamespaceDiscovery: false
```

Comment:
I am wondering if this flag should be defaulted as true to optimize for user experience and document that it can be disabled if you don't like the clusterwide permissions or whatnot.

I am in general advocate for "degradation" experience last.

Reply:
yes, I just didn't want to include a major version change but I am fine if you guys think so.